updating/troubleshooting_updates/gathering-data-cluster-update.adoc

```diff
@@ -0,0 +1,17 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="gathering-data-cluster-update"]
+= Gathering data about your cluster update
+include::_attributes/common-attributes.adoc[]
+:context: troubleshooting_updates
+
+toc::[]
+
+When reaching out to support for issues with an update, it is important to provide data for the support team to use for troubleshooting your failed cluster update.
```

Comment:
Elsewhere in the PR they're referred to as "Red Hat support", so suggest doing so for at least this first instance here as well:
```suggestion
When reaching out to Red Hat support for issues with an update, it is important to provide data for the support team to use for troubleshooting your failed cluster update.
```
I notice also that it's init-cap'd to "Red Hat Support" in the section titles. Idk if that means it should be "Support" throughout this PR as well or not, or if it matters. FWIW, I see the RH SSG has it as "Red Hat Support" in a few places. 🤷